src/pages-and-resources/discussions/app-config-form/apps/shared/ExternalDocumentations.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Hyperlink, MailtoLink,
+} from '@edx/paragon';
+
+import messages from '../lti/messages';
+
+const messageFormatting = (title, instructionType, intl, documentationUrls) => (
+  documentationUrls[instructionType]
+  && instructionType !== 'email_id' && (
+    <div key={instructionType}>
+      <FormattedMessage
+        {...messages.linkText}
+        values={{
+          link: (
+            <Hyperlink
+              destination={documentationUrls[instructionType]}
+            >
+              <FormattedMessage
+                {...messages[instructionType]}
+                values={{
+                  title,
+                }}
+              />
+            </Hyperlink>
+          ),
+        }}
+      />
+    </div>
+  )
+);
+
+function ExternalDocumentations({
+  app, intl, title,
```

Comment:
Why this component needs the full object of `app`? I can see it only needs `app.documentationUrls`

Reply:
yes updated the component.